crates/release_plz_core/src/git/backend.rs

```diff
@@ -156,10 +170,10 @@ pub struct GitLabAuthor {
 
 impl From<GitPr> for GitLabMr {
     fn from(value: GitPr) -> Self {
-        let description = match value.body {
-            Some(d) => d,
-            None => "".to_string(),
-        };
+        let mut desc = "".to_string();
+        if let Some(body) = value.body {
+            desc = body;
+        }
```

Comment:
this is just a refactoring, right?

If so, we could use 

```
        let desc = value.body.unwrap_or_default();
```

it should be equivalent, right?

Reply:
Yes Just a refactor and i have changed it to  `let desc = value.body.unwrap_or_default();` that looks more idiomatic

Reply:
nevermind, I pushed that change already 👍
Sorry if I also left this comment. But it's good that you checked that this refactor looks correct to you as well 👍